thetower/dtower/tourney_results/admin.py

```diff
@@ -275,13 +274,6 @@ class NameDayWinnerAdmin(SimpleHistoryAdmin):
         "nameday_theme",
     )
 
-    def formfield_for_foreignkey(self, db_field, request, **kwargs):
```

Comment:
I don't remember what this did, either we remove entire nameday thing, or let's keep it for now. But I'm pretty sure we remove the entire nameday thing, cause nobody was interested and with Ice quitting, nobody will likely get interested.

Reply:
keeping for now.